cmd/pdcli/alerts/cmd.go

```diff
@@ -0,0 +1,298 @@
+/*
+Copyright © 2021 Red Hat, Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alerts
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/olekukonko/tablewriter"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/constants"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
+	"github.com/spf13/cobra"
+)
+
+type alert struct {
+	incidentID string
+	name       string
+	clusterID  string
+	severity   string
+	status     string
+}
+
+var args struct {
+	high       bool
+	low        bool
+	assignment string
+	columns    string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "alerts",
+	Short: "This command will list all the open alerts. \nIt will list all open high alerts assigned to self by default.",
+	RunE:  alertsHandler,
+}
+
+func init() {
+
+	// Urgency
+	Cmd.Flags().BoolVar(&args.low, "low", false, "view all low alerts")
+	Cmd.Flags().BoolVar(&args.high, "high", true, "view all high alerts")
+
+	// Incident Assignment
+	Cmd.Flags().StringVar(
+		&args.assignment,
+		"assigned-to",
+		"self",
+		"view alerts assigned to either self|team|silentTest",
```

Comment:
There's [OSD-8312](https://issues.redhat.com/browse/OSD-8312) for updating the docs for this feature which I will work on once it has been merged.

Reply:
`❯ pdcli alerts -h
This command will list all the open alerts. 
It will list all open high alerts assigned to self by default.

Usage:
  pdcli alerts [flags]

Flags:
      --assigned-to string   view alerts assigned to either self|team|silentTest (default "self")
      --columns string       Specify which columns to display separated by commas without any space in between (default "incident.id,name,cluster.id,status,severity")
  -h, --help                 help for alerts
      --high                 view all high alerts (default true)
      --low                  view all low alerts
`
Could you please elaborate on what I could add to the help text?

Reply:
This part should be updated as part of OSD-8312 implementation, it's easier to see what changes that way

Reply:
How about `view alerts assigned to a user or a team`?